src/components/com_tjdashboard/administrator/languages/en-GB/en-GB.com_tjdashboard.ini

```diff
@@ -65,7 +65,7 @@ COM_TJDASHBOARD_LIST_VIEW_CREATEDBY = "Created By"
 ;Add/Edit Form View
 COM_TJDASHBOARD_PAGE_ADD_DASHBOARD  = "Dashboards: New"
 COM_TJDASHBOARD_PAGE_EDIT_DASHBOARD = "Dashboards: Edit"
-COM_TJDASHBOARD_PAGE_VIEW_DASHBOARD = "Dashboards: View"
+COM_TJDASHBOARD_PAGE_VIEW_DASHBOARD = "TjDashboard"
```

Comment:
@shitalkakade24 Name should be uniform across the extension

cc
@twsvaishali

Reply:
as gives title "TjDashboard" is correct